src/base-element.js

```diff
@@ -802,6 +806,16 @@ export class BaseElement {
   }
 
   /**
+   * Called when an attribute's value changes.
+   * @param {!string} unusedName
+   * @param {?string} unusedOldValue
+   * @param {?string} unusedNewValue
+   */
+  attributeChangedCallback(unusedName, unusedOldValue, unusedNewValue) {
```

Comment:
Hmm, good point. Yea, internal vsync is probably the most robust (e.g. if services other than Bind mutate element attributes). I can also revisit this later and change it to pass a map of attributes to old/new values if that looks more attractive as Bind support rolls out.